pkg/util/templates/templates.go

```diff
@@ -86,9 +91,18 @@ type multiWindowMultiBurnAlertRule struct {
 			 targeturl: getmeright
 */
 func (r *multiWindowMultiBurnAlertRule) render(url, percent, label, alertName string) monitoringv1.Rule {
-	alertString := "1-avg_over_time(probe_success{" + label + "}[" + r.shortWindow + "]) > (" + r.burnRate + "*(1-" + percent + "))\n" +
-		"and\n" +
-		"1-avg_over_time(probe_success{" + label + "}[" + r.longWindow + "]) > (" + r.burnRate + "*(1-" + percent + " ))"
+	sWindow, _ := time.ParseDuration(r.shortWindow)
+	lWindow, _ := time.ParseDuration(r.longWindow)
+	mPeriod, _ := time.ParseDuration(serviceMonitorPeriod)
```

Comment:
is it safe to ignore the errors returned?

Reply:
I would say so. These values are only set by us, and an error doesn't need to be necessarily handled here as one would fail the unit or integration test and shouldn't be merged anyways